src/werkzeug/formparser.py

```diff
@@ -392,43 +343,9 @@ def __init__(
         assert buffer_size >= 1024, "buffer size has to be at least 1KB"
```

Comment:
Are either of these asserts necessary now?

Reply:
Nope, I've removed them. (New way of parsing ensures that headers if large will be parsed)